### ISSUE TITLE
[Accessibility] Missing the "Items" list level for DomainUpDown control in the inspect tool

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#4207]
* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20559.3

* Have you experienced this same bug with .NET Framework?: 
No

**More info**

1. The issue also can reproduce on .NET 5.0 RC2.
2. The issue can't reproduce on .NET SDK 3.1 and .NET 4.8.

**Problem description:**
Missing the "Items" list level for DomainUpDown control in the inspect tool with UI Automation mode.

**Minimal repro:**

1. Create a .NET Core project with a DomainUpDown control.
2. Build and run the core project.
3. Observe the DomainUpDown control in the Inspect tool on UI Automation mode.

Or

1. Open the attached project.
2. Build and run the core project.
3. Observe the domainUpDown1 control in the Inspect tool on UI Automation mode.
[WindowsFormsApp6.zip](https://github.com/dotnet/winforms/files/5516503/WindowsFormsApp6.zip)

**Expected behavior:**
It displays the correct levels for domainUpDown1 control in the Inspect tool.
![image](https://user-images.githubusercontent.com/45864649/98657047-673e8700-237c-11eb-87e1-a87e70597443.png)

**Actual Behavior:**
Missing the "linkLabel1 link" level for domainUpDown1 control in the inspect tool.
![image](https://user-images.githubusercontent.com/45864649/98657393-ca301e00-237c-11eb-9d5f-bb7e24049802.png)

**Comment by ArtemTatarinov** [comment]
After discussion with @merriemcgaw we decided to close this issue. The Items list of the DomainUpDown control couldn't be visible at UI, there's no way to show them as a dropdown list, etc. The user cannot interact with them as a list from UI, and then they aren't needed in the control's hierarchy shown by the Inspect. So, not showing the "Items" list in the Inspect tool is a new, but expected behavior. The current item of DomaunUpDown is still visible at the Inspect:
![image](https://user-images.githubusercontent.com/87859299/137075819-4b6595bc-b44f-467e-83df-b1e92da744f8.png)
